pkg/xds/envoy/listeners/configurers.go

```diff
@@ -25,10 +25,11 @@ func Kafka(statsName string) FilterChainBuilderOpt {
 	})
 }
 
-func Tracing(backend *mesh_proto.TracingBackend) FilterChainBuilderOpt {
+func Tracing(backend *mesh_proto.TracingBackend, service string) FilterChainBuilderOpt {
```

Comment:
I think some godoc comments describing the semantics of the "service" parameter would be helpful. I had to trace it through to `envoy_trace.DatadogConfig` and then look up the envoy reference docs to know what it was for :)

Reply:
OK, but since this is just a tiny function that emits the struct, I think it makes more sense to document its meaning centrally in the struct itself.